app/containers/EmojiPicker/EmojiCategory.tsx

```diff
@@ -26,48 +28,44 @@ const renderEmoji = (emoji: IEmoji, size: number, baseUrl: string) => {
 	);
 };
 
-class EmojiCategory extends React.Component<IEmojiCategory> {
-	renderItem(emoji: IEmoji) {
-		const { baseUrl, onEmojiSelected } = this.props;
-		return (
+const EmojiCategory = React.memo(
```

Comment:
Same as the comment I left on the other PR.
`React.memo` is possibly making the performance worse here.
`onEmojiSelected` might change over re-renders and it's an inline function, which is going to passed as a forever changing prop.
If that's true, the component is always re-rendering anyway :)